### PR TITLE
Fixed a few bugs with access levels

### PR DESCRIPTION
**Pull request by andrii52**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by OleksandrBilan — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

File: src/pages/City/City/CityDocuments.tsx [hunk 1]
```diff
@@ -27,6 +27,9 @@ const CityDocuments = () => {
       setLoading(true);
       try {
         const response = await getCityById(+id);
+        const response1 = await cityNameOfApprovedMember(userApi.getActiveUserId());
```
Comment (ostappppppp) [inline src/pages/City/City/CityDocuments.tsx:30]: Please, provide an appropriate name
This name doesn't meet with conventions

File: src/pages/Club/Club/Club.tsx [hunk 2]
```diff
@@ -164,8 +164,8 @@ const Club = () => {
 
     try {
       const response = await getClubById(+id);
-      const responce1 = await clubNameOfApprovedMember(userApi.getActiveUserId());
-      setActiveUserClub(responce1.data);
+      const response1 = await clubNameOfApprovedMember(userApi.getActiveUserId());
```
Comment (ostappppppp) [inline src/pages/Club/Club/Club.tsx:167]: The same here

File: src/pages/Club/Club/ClubDocuments.tsx [hunk 3]
```diff
@@ -2,28 +2,49 @@ import React, {useEffect, useState} from 'react';
...
+    const [club, setClub] = useState<ClubProfile>(new ClubProfile());
     const [loading, setLoading] = useState<boolean>(false);
+    const [activeUserRoles, setActiveUserRoles] = useState<string[]>([]);
+    
+
+    const getClub = async () => {
+      setLoading(true);
+      try {
+        const response = await getClubById(+id);
+        const response1 = await clubNameOfApprovedMember(userApi.getActiveUserId());
```
Comment (ostappppppp) [inline src/pages/Club/Club/ClubDocuments.tsx:32]: Also here

File: src/pages/City/City/CityDocuments.tsx (hunk 1 above)
Comment (andrii52) [reply src/pages/City/City/CityDocuments.tsx]: Changed this

**Review by sharix4 — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Pull request merged**